### PR TITLE
Release 1.3.0

### LINKED ISSUE #2706: Streamlink 1.3.0
Is there anything which is blocking a new release or needs to get merged?

I would really love to publish the Twitch plugin fix. In the PR, I've said that we should probably add some notes to the documentation of the now useless Twitch OAuth parameters, but I think it's not really important. I will add a comment with a full overview of the issue to the locked thread later today. I didn't have the time yesterday.

### PR DESCRIPTION
**Pull request by bastimeyer**
Closes #2706

If there's anything that I've missed or should be changed/reworded, don't hesitate to tell me.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,78 @@
 # Changelog
 
+## streamlink 1.3.0 (2019-11-21)
+
+A new release with plugin updates and fixes, including Twitch.tv (see #2680), which had to be delayed due to back and forth API changes.
```
Comment (gravyboat) [inline CHANGELOG.md]: I don't think the changelog links directly to issues right? So it needs to be `(see [#2680](https://github.com/streamlink/streamlink/issues/2680))` for any references to the issue.

Comment (bastimeyer) [reply CHANGELOG.md]: The github release pages use the same github flavored markdown as comments on issue trackers and issue- or commit-ids will get turned into links automatically. The rendered markdown on the git tree however does not.

Let me change this real quick...

**Comment by bastimeyer** [comment]
Let me change the date before merging this

**Comment by bastimeyer** [comment]
I'm going to bed now. To the one who merges this, please don't forget to add/push the `1.3.0` tag afterwards, so that the release can be built. If further changes are needed, just add them to the PR branch.

**Comment by gravyboat** [comment]
Merged, [tagged, and pushed as 1.3.0](https://github.com/streamlink/streamlink/tree/1.3.0). Thanks @bastimeyer!

**Pull request merged**